### PR TITLE
Remove pipecd_server prefixes from incoming request dashboard

### PR DESCRIPTION
**Pull request by nakabonne**
**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

Fixes #

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

### CODE REVIEW

File: manifests/pipecd/grafana-dashboards/control-plane/incoming-request.json [hunk 1]
```diff
@@ -777,5 +777,5 @@
   "timezone": "",
   "title": "Incoming Request",
   "uid": "3jeba3R7z",
-  "version": 16
+  "version": 4
```
Comment (khanhtc1202) [inline manifests/pipecd/grafana-dashboards/control-plane/incoming-request.json:780]: not sure about downgrade the version 🤔

Comment (nakabonne) [reply manifests/pipecd/grafana-dashboards/control-plane/incoming-request.json:780]: Sometimes it happens and I'm sure it doesn't matter

Comment (khanhtc1202) [reply manifests/pipecd/grafana-dashboards/control-plane/incoming-request.json:780]: Okay 🙌

**Comment by khanhtc1202** [comment]
Way to go
/approve

File: manifests/pipecd/grafana-dashboards/control-plane/incoming-request.json (hunk 1 above)
Comment (nakabonne) [reply manifests/pipecd/grafana-dashboards/control-plane/incoming-request.json:780]: I mean the version seems to be reset when reload by ConfigMap, that's why it happens

**Pull request merged**